docs/example_NIR_cube.rst

```diff
@@ -0,0 +1,97 @@
+Fitting H2 1-0 S(1) line in NIR data cube
+================================
+.. code-block:: python
+
+  import pyspeckit
+  from pylab import *
+  import numpy as np
+  import astropy.io.fits as pyfits
+
+
+  # load the cube as a pyspeckit Cube
+  # I usually get an error here if cube.fits' header doesn't have 'CTYPE3' = 'WAVE'
+  cube = pyspeckit.Cube('cube.fits')
+
+  # Slice the cube over the wavelength range you'd like to fit
+  cube_h2 = cube.slice(21145,21250,unit='Angstrom')
+
+  # rescale data to make fitting nicer
+  # I add an offset to avoid negative values
+  cube_h2.cube *= 1e17 
+  cube_h2.cube += 30.
+
+  # Do an initial plot & fit of a single spectrum
+  # at a pixel with good S/N
+  cube_h2.plot_spectrum(100,100)
+
+  # Here I'm fitting two gaussians with 4 parameters each (background offset, 
+  # amplitude, wavelength centroid, linewidth).
+  # I find that if I let both backgrounds be free parameters, pyspeckit returns
+  # unrealistic values for both backgrounds, so I fix the 2nd gaussian's background
+  # level to 0.
```

Comment:
Ah, I see it is in fiteach.  Maybe just note that here?  And note that this is just a setup command; it's not really doing anything meaty.